src/components/Buttons.jsx

```diff
@@ -0,0 +1,122 @@
+import React, { useCallback, useMemo } from "react";
+import { ToggleButton, ToggleButtonGroup } from "@mui/material";
+import {
+  ViewStream,
+  Subject,
+  Edit,
+  Preview,
+  Undo,
+  Redo, Save
```

Comment:
@superdav42 could add tool tips on these. Some are more international than others. At some point, we should consider some localizations on the tool tips, but that can wait.